webservices/common/models/costs.py

```diff
@@ -58,9 +63,9 @@ class Electioneering(db.Model):
     disbursement_amount = db.Column('reported_disb_amt', db.Numeric(30, 2), index=True)
     purpose_description = db.Column('disb_desc', db.String)
     report_year = db.Column('rpt_yr', db.Integer, index=True)
-
-    committee = utils.related_committee('committee_id')
-    candidate = utils.related_candidate('candidate_id')
+    election_type = db.Column('election_tp', db.String)
```

Comment:
What do you think about displaying them as just the election in the API? We already have several dates on this endpoint, so year is covered

Thanks again!